### PR TITLE
[Routing] Service route loaders tag

### PR DESCRIPTION
**Pull request by fancyweb**
#eufossa 

doc for https://github.com/symfony/symfony/pull/32598

### CODE REVIEW

**Comment by wouterj** [comment]
Seems like you changed the single to double colon thing in both of your PRs. Can you remove it from one PR? (and if it's deprecated in 3.4 or 4.2, maybe create a new PR with that change based on the correct branch?)

File: routing/custom_route_loader.rst [hunk 1]
```diff
@@ -119,14 +119,20 @@ and configure the service and method to call:
...
+            $routes->import('admin_route_loader::loadRoutes', 'service');
         };
 
 In this example, the routes are loaded by calling the ``loadRoutes()`` method
 of the service whose ID is ``admin_route_loader``. Your service doesn't have to
 extend or implement any special class, but the called method must return a
 :class:`Symfony\\Component\\Routing\\RouteCollection` object.
 
+If you're using :ref:`autoconfigure <services-autoconfigure>`, your service should
+implement the `ServiceRouterLoaderInterface` interface to be tagged automatically by
```
Comment (wouterj) [inline routing/custom_route_loader.rst]: inline code has to be surrounded by double backticks in reStructured Text: ``` ``ServiceRouterLoaderInterface`` ```

File: routing/custom_route_loader.rst [hunk 2]
```diff
@@ -119,14 +119,20 @@ and configure the service and method to call:
...
+Symfony!
```
Comment (wouterj) [inline routing/custom_route_loader.rst]: I think we can make this less expressive and replace `!` with `.`. Can you also add the next sentence directly to this paragraph, to save some space?

Comment (OskarStark) [reply routing/custom_route_loader.rst]: I would remove the “by Symfony!” Completely and end with a dot

**Comment by fancyweb** [comment]
@wouterj it has been deprecated since 4.1. I will create another MR for a fix on 4.2.

**Review by wouterj — APPROVED** [review]

**Comment by fancyweb** [comment]
The code has been approved but with a change in the tag name. I just updated the doc accordingly.

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
@fancyweb does it now belong to: https://github.com/symfony/symfony/pull/32582 ?

**Comment by fancyweb** [comment]
No it's for https://github.com/symfony/symfony/pull/32598

**Comment by OskarStark** [comment]
Ok, thank you, so the label `Waiting Code Merge` is still valid 👍

**Comment by wouterj** [comment]
Thanks @fancyweb for documenting your code changes! I've added a minor deprecated directive in https://github.com/symfony/symfony-docs/commit/6d4b9f20ce435c86cc59611e7cdb662db0b065bb to highlight better that something changed here.

Also, I had a small fight with our merge tool, so we've lost the merge commit (but not your changes, so all is good).

**Pull request closed without merging**